src/graph/executor/algo/IsomorExecutor.cpp

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2020 vesoft inc. All rights reserved.
+//
+// This source code is licensed under Apache 2.0 License.
+
+#include "graph/executor/algo/IsomorExecutor.h"
+
+#include "graph/planner/plan/Algo.h"
+
+namespace nebula {
+namespace graph {
+
+folly::Future<Status> IsomorExecutor::execute() {
+   // TODO: Replace the following codes with subgraph matching. Return type.
+   SCOPED_TIMER(&execTime_);
+   auto* subgraph = asNode<Subgraph>(node());
```

Comment:
Just to make sure you are going to replace `Subgraph` to `Isomor` plan node right?

Reply:
This is the old version changed from the SubgraphExecutor which I need to change later against with adding the subgraph.cpp (subgraph matching algorithm).